Compile Errors

Hi Mario

I loaded the core project, executed "CUBA > Advanced > Install App Component" and the core project compiled successfully. Afterwards I loaded the lancester-pet-clinic and tried to run the project in the IDE. I got the error messages below. Any idea what went wrong? Thenaks for a feedback.
Werner

 
Execution failed for task ':lancasterpetclinic-web:compileJava'.
> Could not resolve all files for configuration ':lancasterpetclinic-web:compileClasspath'.
   > Could not find com.haulmont.sample.petclinic:petclinic-web-themes:2.0.1-SNAPSHOT.
     Searched in the following locations:
       - file:/Users/spadmin/.m2/repository/com/haulmont/sample/petclinic/petclinic-web-themes/2.0.1-SNAPSHOT/maven-metadata.xml
       - file:/Users/spadmin/.m2/repository/com/haulmont/sample/petclinic/petclinic-web-themes/2.0.1-SNAPSHOT/petclinic-web-themes-2.0.1-SNAPSHOT.pom
       - file:/Users/spadmin/.m2/repository/com/haulmont/sample/petclinic/petclinic-web-themes/2.0.1-SNAPSHOT/petclinic-web-themes-2.0.1-SNAPSHOT.jar
       - https://dl.bintray.com/cuba-platform/main/com/haulmont/sample/petclinic/petclinic-web-themes/2.0.1-SNAPSHOT/maven-metadata.xml
       - https://dl.bintray.com/cuba-platform/main/com/haulmont/sample/petclinic/petclinic-web-themes/2.0.1-SNAPSHOT/petclinic-web-themes-2.0.1-SNAPSHOT.pom
       - https://dl.bintray.com/cuba-platform/main/com/haulmont/sample/petclinic/petclinic-web-themes/2.0.1-SNAPSHOT/petclinic-web-themes-2.0.1-SNAPSHOT.jar
       - https://jcenter.bintray.com/com/haulmont/sample/petclinic/petclinic-web-themes/2.0.1-SNAPSHOT/maven-metadata.xml
       - https://jcenter.bintray.com/com/haulmont/sample/petclinic/petclinic-web-themes/2.0.1-SNAPSHOT/petclinic-web-themes-2.0.1-SNAPSHOT.pom
       - https://jcenter.bintray.com/com/haulmont/sample/petclinic/petclinic-web-themes/2.0.1-SNAPSHOT/petclinic-web-themes-2.0.1-SNAPSHOT.jar
     Required by:
         project :lancasterpetclinic-web

Should I wait for fix in the project as mentioned above? 
If I follow the instructions I am getting:

pwd
/Users/spadmin/Downloads/cuba-petclinic-product-customizations-master/lancaster-petclinic
spadmins-MBP:lancaster-petclinic spadmin$ 
git reset --hard
fatal: not a git repository (or any of the parent directories): .git
spadmins-MBP:lancaster-petclinic spadmin$ 
./gradlew install
> Task :lancasterpetclinic-web:compileJava FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':lancasterpetclinic-web:compileJava'.
> Could not resolve all files for configuration ':lancasterpetclinic-web:compileClasspath'.
   > Could not find com.haulmont.sample.petclinic:petclinic-web-themes:2.0.1-SNAPSHOT.
     Searched in the following locations:
       - file:/Users/spadmin/.m2/repository/com/haulmont/sample/petclinic/petclinic-web-themes/2.0.1-SNAPSHOT/maven-metadata.xml
       - file:/Users/spadmin/.m2/repository/com/haulmont/sample/petclinic/petclinic-web-themes/2.0.1-SNAPSHOT/petclinic-web-themes-2.0.1-SNAPSHOT.pom
       - file:/Users/spadmin/.m2/repository/com/haulmont/sample/petclinic/petclinic-web-themes/2.0.1-SNAPSHOT/petclinic-web-themes-2.0.1-SNAPSHOT.jar
       - https://dl.bintray.com/cuba-platform/main/com/haulmont/sample/petclinic/petclinic-web-themes/2.0.1-SNAPSHOT/maven-metadata.xml
       - https://dl.bintray.com/cuba-platform/main/com/haulmont/sample/petclinic/petclinic-web-themes/2.0.1-SNAPSHOT/petclinic-web-themes-2.0.1-SNAPSHOT.pom
       - https://dl.bintray.com/cuba-platform/main/com/haulmont/sample/petclinic/petclinic-web-themes/2.0.1-SNAPSHOT/petclinic-web-themes-2.0.1-SNAPSHOT.jar
       - https://jcenter.bintray.com/com/haulmont/sample/petclinic/petclinic-web-themes/2.0.1-SNAPSHOT/maven-metadata.xml
       - https://jcenter.bintray.com/com/haulmont/sample/petclinic/petclinic-web-themes/2.0.1-SNAPSHOT/petclinic-web-themes-2.0.1-SNAPSHOT.pom
       - https://jcenter.bintray.com/com/haulmont/sample/petclinic/petclinic-web-themes/2.0.1-SNAPSHOT/petclinic-web-themes-2.0.1-SNAPSHOT.jar
     Required by:
         project :lancasterpetclinic-web

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 6.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/5.6.4/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 3s
17 actionable tasks: 6 executed, 11 up-to-date
spadmins-MBP:lancaster-petclinic spadmin$

--- a/petclinic-core/settings.gradle
+++ b/petclinic-core/settings.gradle
@@ -1,6 +1,6 @@
 rootProject.name = 'cuba-petclinic'
 def modulePrefix = 'petclinic'
-include(":${modulePrefix}-global", ":${modulePrefix}-core", ":${modulePrefix}-web", ":${modulePrefix}-web-themes")
+include(":${modulePrefix}-global", ":${modulePrefix}-core", ":${modulePrefix}-web")
 project(":${modulePrefix}-global").projectDir = new File(settingsDir, 'modules/global')
 project(":${modulePrefix}-core").projectDir = new File(settingsDir, 'modules/core')
 project(":${modulePrefix}-web").projectDir = new File(settingsDir, 'modules/web')

